sql/hive/src/test/scala/org/apache/spark/sql/hive/orc/HiveOrcSourceSuite.scala

```diff
@@ -156,28 +156,6 @@ class HiveOrcSourceSuite extends OrcSuite with TestHiveSingleton {
         sql("select testType()").write.mode("overwrite").orc(orcDir)
       }.getMessage
       assert(msg.contains("ORC data source does not support calendarinterval data type."))
-
-      // read path
```

Comment:
In read path, ORC should support `CalendarIntervalType` and `NullType`.

Reply:
Is there any write path test already?

Reply:
@HyukjinKwon No, the unit test is about unsupported data types, and ORC supports all data types in read path.

Reply:
I mean the tests were negative tests. so I was expecting that we'd have positive tests.

Reply:
Spark can never write out interval type, do we really need to support interval type at read path?

Reply:
+1 for ^